### PR TITLE
Fix issues with systemd images

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: systemd/Dockerfile.wheezy.tpl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM #{FROM}
+
+# Install Systemd
+
+RUN echo "deb http://ftp.debian.org/debian/ wheezy-backports main contrib non-free" >> /etc/apt/sources.list
+RUN apt-get update && apt-get -t wheezy-backports install -y systemd \
```
Comment (Page-) [inline systemd/Dockerfile.wheezy.tpl]: What is `-t`?

File: systemd/generate-dockerfile.sh [hunk 2]
```diff
@@ -19,8 +20,20 @@ for repo in $repos; do
 	for suite in $suites; do
 		dockerfilePath=$repo/$suite
 		mkdir -p $dockerfilePath
-		sed -e s~#{FROM}~resin/$baseImage:$suite~g Dockerfile.tpl > $dockerfilePath/Dockerfile
-		cp entry.sh launch.service $dockerfilePath/
+		
+		if [ $suite == 'wheezy' ] && [ $repo != 'rpi' ]; then
```
Comment (Page-) [inline systemd/generate-dockerfile.sh]: Can we have comments explaining why the special cases

File: systemd/Dockerfile.wheezy.tpl (hunk 1 above)
Comment (nghiant2710) [inline systemd/Dockerfile.wheezy.tpl]: -t to retrieve packages from wheezy-backports repository. http://manpages.ubuntu.com/manpages/lucid/man8/apt-get.8.html

File: device-base/generate-dockerfile.sh [hunk 3]
```diff
@@ -21,21 +35,34 @@ for device in $devices; do
...
 
-	# Only for armv7hf devices
-	if [ $device == 'raspberrypi2' ] || [ $device == 'beaglebone' ]; then
-		mkdir -p $dockerfilePath/sid
-		sed -e s~#{FROM}~resin/$baseImage:sid~g \
-			-e s~#{SUITE}~sid~g $template > $dockerfilePath/sid/Dockerfile		
-	fi
+			case "$suite" in
+			'wheezy')
+				sourcelist="$bb_sourceslist_wheezy_cmd"
```
Comment (Page-) [inline device-base/generate-dockerfile.sh]: You can use:
```
sourcelist=bb_sourceslist_$suite_cmd
${!sourcelist}
```

File: systemd/Dockerfile.wheezy.tpl (hunk 1 above)
Comment (Page-) [inline systemd/Dockerfile.wheezy.tpl]: Can you use `--target-release` please

File: device-base/generate-dockerfile.sh (hunk 3 above)
Comment (nghiant2710) [inline device-base/generate-dockerfile.sh]: @Page- ,this doesn't work for me

File: systemd/Dockerfile.wheezy.tpl (hunk 1 above)
Comment (nghiant2710) [inline systemd/Dockerfile.wheezy.tpl]: Updated!

File: device-base/generate-dockerfile.sh (hunk 3 above)
Comment (Page-) [inline device-base/generate-dockerfile.sh]: What did you try? You would have to replace the `$sourcelist` uses with `${!sourcelist}` - it's not a big deal though as I'm not sure dropping the switch to use that style would make the code any clearer anyway

Comment (nghiant2710) [inline device-base/generate-dockerfile.sh]: Oops, I get it wrong, it works. Btw I agree that this change won't make it look better.

**Pull request merged**